Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -814,7 +832,57 @@ - (void)blockBannerViewWasTapped:(UIGestureRecognizer *)sender
 - (void)noLongerVerifiedBannerViewWasTapped:(UIGestureRecognizer *)sender
 {
     if (sender.state == UIGestureRecognizerStateRecognized) {
-        [self showConversationSettingsAndShowVerification:YES];
+        UIAlertController *actionSheetController =
+            [UIAlertController alertControllerWithTitle:nil
+                                                message:nil
+                                         preferredStyle:UIAlertControllerStyleActionSheet];
+
+        __weak MessagesViewController *weakSelf = self;
+        UIAlertAction *unblockAction = [UIAlertAction
```

Comment:
a little confused, but I think it's just a failure to update variable names after copy/paste.

This should be something like `verifyPrivacyAction` not `unblockAction` right?